src/PrestaShopBundle/Form/Admin/Sell/Order/Invoices/InvoiceOptionsDataProvider.php

```diff
@@ -40,21 +40,13 @@ final class InvoiceOptionsDataProvider implements FormDataProviderInterface
      */
     private $invoiceOptionsConfiguration;
 
-    /**
-     * @var int
-     */
-    private $nextInvoiceNumber;
-
     /**
      * @param DataConfigurationInterface $invoiceOptionsConfiguration
-     * @param int $nextInvoiceNumber next available invoice number
      */
     public function __construct(
-        DataConfigurationInterface $invoiceOptionsConfiguration,
-        $nextInvoiceNumber
```

Comment:
Can we keep this data, but not use it? It will be a useless constructor param but this way it's not a BC break. Avoiding a BC break is good if the cost is little 😊

Reply:
Alright, makes sense.